packages/cozy-harvest-lib/src/components/EditAccountModal.jsx

```diff
@@ -23,66 +23,75 @@ import { withMountPointPushHistory } from './MountPointContext'
 import logger from '../logger'
 import { withTracker } from './hoc/tracking'
 import useTimeout from './hooks/useTimeout'
+import { withRouter } from 'react-router'
 
 const showStyle = { opacity: 1, transition: 'opacity 0.3s ease' }
 const hideStyle = { opacity: 0, transition: 'opacity 0.3s ease' }
 
-const DumbEditAccountModal = ({
-  konnector,
-  account,
-  trigger,
-  fetching,
-  redirectToAccount
-}) => {
-  const { dialogProps, dialogTitleProps } = useCozyDialog({
-    open: true,
-    size: 'm',
-    onClose: redirectToAccount
-  })
+const DumbEditAccountModal = withRouter(
```

Comment:
Nit: Have you tried to use `useLocation` ? This way there is no HOC involved.

Reply:
useLocation doest not work

Reply:
Do you know why ?